### PR TITLE
Several misc fixes

### PR DESCRIPTION
**Pull request by R-L-T-Y**
- fix typo to show forced state torrents
- add function and button for torrents to force seeding/downloading (#33)
- add function and button for qBittorrent to shut down safely for nox users
- stop confirming torrents reannouncing, it's not necessary
- use official syntax of protocolHandler (from `qb-web://#url=%magnet%` to `qb-web://#download=%magnet%` ), since there are currently no button to register right now

### CODE REVIEW

File: src/locale/en.ts [hunk 1]
```diff
@@ -23,7 +23,7 @@ export default {
   refresh: 'Refresh',
   location: 'Location',
   rename: 'Rename',
-
+  triggerApplicationShutdown: 'Exit qBittorrent',
```
Comment (CzBiX) [inline src/locale/en.ts:26]: `trigger_application_shutdown`

File: src/locale/zh-CN.ts [hunk 2]
```diff
@@ -23,7 +23,7 @@ export default {
   refresh: '刷新',
   location: '位置',
   rename: '重命名',
-
+  triggerApplicationShutdown: '退出qBittorrent',
```
Comment (CzBiX) [inline src/locale/zh-CN.ts:26]: `trigger_application_shutdown`

File: src/Api.ts [hunk 3]
```diff
@@ -38,6 +38,10 @@ class Api {
     return this.axios.get('/app/preferences');
   }
 
+  public shutdownApplication() {
+    return this.axios.get('/app/shutdown');
```
Comment (CzBiX) [inline src/Api.ts:42]: Should be post?

File: src/components/drawer/DrawerFooter.vue [hunk 4]
```diff
@@ -56,6 +56,13 @@
       >
         <v-icon v-text="darkModeIcon" />
       </v-btn>
+      <v-btn
+        icon
+        @click="triggerApplicationShutdown"
+        :title="$t('triggerApplicationShutdown')"
+      >
+        <v-icon> mdi-power-plug-off</v-icon>
```
Comment (CzBiX) [inline src/components/drawer/DrawerFooter.vue:64]: trim space

File: src/Api.ts (hunk 3 above)
Comment (R-L-T-Y) [reply src/Api.ts:42]: I don't see any specification in the [docs](https://github.com/qbittorrent/qBittorrent/wiki/WebUI-API-(qBittorrent-4.1)#shutdown-application), GET works without any problem

**Comment by R-L-T-Y** [comment]
Thank you for your speedy review!

**Comment by CzBiX** [comment]
Merged, thanks.

**Pull request closed without merging**